middleware/jwt.go

```diff
@@ -32,6 +32,13 @@ type (
 		// ErrorHandlerWithContext is almost identical to ErrorHandler, but it's passed the current context.
 		ErrorHandlerWithContext JWTErrorHandlerWithContext
 
+		// NoErrorContinuesExecution allows next middleware/handler to be called when ErrorHandlerWithContext decides to
+		// swallow the error (returns nil).
+		// This is useful in cases when portion of your site/api is publicly accessible and has extra features for authorized
+		// users. In that case you can use ErrorHandlerWithContext to set default public JWT token value to request and
+		// continue with handler chain. Assuming logic downstream execution chain has to check that (public) token value.
```

Comment:
Alternative name might be `ContinueOnIgnoredError`

Let me try to rephrase the comment (hope I understood the usecase):
```
// ContinueOnIgnoredError allows the next middleware/handler to be called when ErrorHandlerWithContext decides to
// ignore the error (by returning `nil`).
// This is useful when parts of your site/api allow public access and some authorized routes provide extra functionality.
// In that case you can use ErrorHandlerWithContext to set a default public JWT token value in the request context 
// and continue. Some logic down the remaining execution chain needs to check that (public) token value then.
```